easybuild/easyconfigs/e/EVidenceModeler/EVidenceModeler-2.1.0-foss-2023a.eb

```
@@ -0,0 +1,42 @@
+easyblock = 'Tarball'
+
+name = 'EVidenceModeler'
+version = '2.1.0'
+
+homepage = 'https://github.com/EVidenceModeler/EVidenceModeler'
+description = """ EVM provides a flexible and intuitive framework for 
+combining diverse evidence types into a single automated gene structure annotation system."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+sources = [{
+    'filename': '%(name)s-v%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/EVidenceModeler',
+        'repo_name': '%(name)s',
+        'tag': '%(name)s-v%(version)s',
+        'recursive': True,
+    }
+}]
+patches = ['EVidenceModeler-2.0.0_set-correct-CFlags-for-ParaFly.patch']
+checksums = [None]
+
+dependencies = [
+    ('PASA', '2.5.3',),
+]
+
+# Install ParaFly
+postinstallcmds = ["cd %(installdir)s && rm plugins/ParaFly/bin/ParaFly && make"]
+
+sanity_check_paths = {
+    'files': ['EVidenceModeler', 'plugins/ParaFly/bin/ParaFly'],
+    'dirs': ['plugins/ParaFly', 'testing'],
+}
+
+modextrapaths = {
+    'EVM_HOME': '',
+}
+
+sanity_check_commands = ["$EVM_HOME/EVidenceModeler -h 2>&1 | grep 'Evidence Modeler'"]
```

@laraPPr Shouldn't we add the top-level installation directory to `$PATH`?
```suggestion
modextrapaths = {
    'EVM_HOME': '',
    'PATH': ''.
}

sanity_check_commands = ["EVidenceModeler -h 2>&1 | grep 'Evidence Modeler'"]

---

I'm not sure funannotate only uses $EVM_HOME and that is also what their own documentation says. But in a sense their is no harm in adding it. I think.